v1/middleware/auth.go

```diff
@@ -124,6 +124,37 @@ func isBanned(ctx *gin.Context) (bool, error) {
 	return true, nil
 }
 
+func isSuspended(ctx *gin.Context, service string) gin.HandlerFunc {
```

Comment:
```suggestion
func IsSuspended(feature string) gin.HandlerFunc {
```
huruf depan functionnya dibuat uppercase biar public. lalu ctx gaperlu dijadiin param